extensions/amp-facebook-page/0.1/amp-facebook-page.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import {getIframe, preloadBootstrap} from '../../../src/3p-frame';
+import {listenFor} from '../../../src/iframe-helper';
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {removeElement} from '../../../src/dom';
+import {dashToUnderline} from '../../../src/string';
+
+class AmpFacebookPage extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?HTMLIFrameElement} */
+    this.iframe_ = null;
+
+    /** @private {string} */
+    this.dataLocale_ = element.getAttribute('data-locale');
```

Comment:
You should assign `this.dataLocale_` as the result of a ternary or `||` expression instead of an `if`, so that you can annotate `this.dataLocale_` as `@const` for more type safety

Reply:
I've made the change here. However, I have written the same code in other files: amp-facebook{,-like,-comments}.js. 

Do you mind if I make the change in a separate PR? Or would you like that done here. Happy either way.

Reply:
Ah cool, a separate PR would be ok